src/components/LightSwitch.js

```diff
@@ -1,11 +1,34 @@
 import React from "react"
+import { useState } from "react"
 
 const LightSwitch = () => {
+  const [lightSwitchStatus, setLightSwitchStatus] = useState(false)
+
+  const turnLight = () => {
+    if (lightSwitchStatus === false){
+      setLightSwitchStatus(true)
+    } else {
+      setLightSwitchStatus(false)
+    }
+  }
+
+  let lightStatus = ""
+  if (lightSwitchStatus === false) {
+    lightStatus = "Off"
+  } else {
+    lightStatus = "On"
+  }
+
   return (
     <>
-      <div className="lightbulb"></div>
+      <div 
+        onClick={turnLight} 
+        className="lightbulb" 
+        style={{backgroundColor: lightSwitchStatus ? 'yellow' : 'white'}}>
```

Comment:
Neat method for switching the background.

Reply:
Great job!  I was surprised to find out we could use conditionals in JSX land.  This is something to consider.

Reply:
I had no idea you could put conditionals in JSX, that's really cool.